README.md

```diff
@@ -43,18 +31,17 @@ git remote add origin <server>
 git push origin main
 ```
 
-# Setting up DevOps pipelines
-
-The cookiecutter template will also generate some DevOps pipelines you can use for linting, unit tests and building your Python package. 
+## DevOps pipelines
 
-For more information on setting up linting and unit test pipelines, see: https://wikiddc.corporateroot.net/doku.php?id=python_linting_and_unit_tests_in_devops_pipelines
+The cookiecutter template includes three DevOps pipelines, included in
```

Comment:
Did you try if the pipelines work on github?

Reply:
Nice, good changes @Gijs-Koot . Have you tested the github cookiecutter + pipeliens? I haven't tried it yet from the github account. Might be good to check if everythign is working, but if not that could also easily be in a different PR.